string measurment
string sensor
int64 value